tuf/ngclient/updater.py

```diff
@@ -316,76 +319,63 @@ def _load_targets(self, role: str, parent_role: str) -> None:
             self._persist_metadata(role, data)
 
     def _preorder_depth_first_walk(
-        self, target_filepath: str
-    ) -> Union[Dict[str, Any], None]:
+        self,
+        target_filepath: str,
+        visited_role_names: Set[str],
+        current_role_pair: List[Tuple[str, ...]],
```

Comment:
What does `...` mean in `List[Tuple[str, ...]]`?

Reply:
annotations for both visited_role_names and current_role_pair are wrong